run_code_quality.m

```diff
@@ -0,0 +1,21 @@
+% Run all tests in current working directory
+%
+% This runs all tests in the current working directory, and produces
+% code coverage files in the "artifacts" directory
+
+import matlab.unittest.TestRunner
+import matlab.unittest.Verbosity
+
+
+oldpath = path;
+path('.', oldpath);
+
+code_quality_path = fullfile(pwd, 'linting', 'code-quality');
+suite = testsuite(code_quality_path, 'IncludeSubfolders', true);
+
+[~,~] = mkdir('artifacts');
```

Comment:
If we're not saving any code coverage output (which we shouldn't with linting!) I don't think this directory needs creating?

Reply:
Yes! Just removed it